data/hai/hai missions.txt

```diff
@@ -1932,3 +1932,86 @@ mission "Station Exploration Abandon Ship"
+	description "Bring a Hai spiritual leader to <destination>. He will pay you <payment>."
+	minor
+	clearance "You inform the authorities of the passenger you are transporting, and they quickly allow you to land."
+	source
+		government "Hai"
+		not attributes "station"
+	destination
+		attributes "unfettered"
+	passengers 1
+	to offer
+		random < 10
+		or
+			has "outfit: Quantum Keystone"
+			has "outfit: Quantum Key Stone"
+		has "First Contact: Hai: offered"
+	on offer
+		conversation
+			`It is an unusually foggy day on <origin>; the air is so thick with moisture that you cannot see more than a few paces in front of you. Perhaps that's why you're so startled when you turn around and see a Hai standing right next to you. He is so close you can see dewdrops on his whiskers, and you immediately notice the prominent jewelry that he is wearing. He appears to be quite old, judging by his stooped-over posture and the white fur around his muzzle.`
+			branch remnant
+				has "outfit: Quantum Key Stone"
+			`	"I see that you are wiser than most humans here," he says, pointing towards your ship. "You carry a Keystone, a shield against the tempest of space. Let no one else tell you otherwise." He stops to catch his breath. "Now, onto more worldly things. My name is Hareek. Are you willing and able to take me to <destination> for <payment>?"`
+				goto choice
+			label remnant
+			`	"I see that you are wiser than most humans here," he says, pointing towards your ship. "You carry a Keystone, a shield against the tempest of space. Let no one else tell you otherwise." He stops to catch his breath. "I have noticed that the Keystone your ship bears is from beyond the wormhole. If your people have access to Keystones as well, then I hope they too become wise to its power. Now, onto more worldly things. My name is Hareek. Are you willing and able to take me to <destination> for <payment>?"`
+			label choice
+			choice
+				`	"I can take you there."`
+					goto agree
+				`	"Why would you want to go to <destination>? Isn't that an Unfettered planet?"`
+					goto ask
+					to display
+						or
+							has "visited system: Wah Ki"
+							has "Wanderers: Translation Machine: offered"
+							has "First Contact: Unfettered: offered"
+				`	"I'm not going there. You'll have to ask someone else."`
+					decline
+			label ask
+			`	The Hai pauses for a moment, as if unsure of what to say. "My teachings and prophecies are more... appreciated among our northern brethren."`
+			choice
+				`	"Teachings?"`
+				`	"Alright, I'll take you there."`
+					goto agree
+				`	"Unfettered space is too dangerous for me. You'll have to find someone else."`
+					goto dangerous
+			label teachings
+			`	An unreadable expression appears on Hareek's face. "Ah, apologies for my rudeness - you probably do not know who I am. How do I explain this... You have your preachers and wise men among the humans, yes? I am something similar. Now, will you take me to <destination>?"`
```

Comment:
```suggestion
			`	An unreadable expression appears on Hareek's face. "Ah, apologies for my rudeness - you probably do not know who I am. How do I explain this... You have your preachers and wise men and women among the humans, yes? I am something similar. Now, will you take me to <destination>?"`
```
Thats more inclusive, right? Mb I'm the only one this slightly bugs

Reply:
Best course for inclusiveness would be to "wise ones," and technically the real meaning is "wise humans" not "wise masculine humans," though there are still gendered connotations to the historic role which make the current language still apt.

If we do change this, "wise ones" or "wise humans" would be best; just repetition of binary genders has mostly phased out as an inclusive ideal, like "person first language."

Reply:
I wanted to say wise humans but its "among the humans" so that'd be repetitive